docs/chapters/architecture/SOTA-system.rst

```diff
@@ -617,13 +617,66 @@ see the status of your rollout campaign changed.
 Going further
 ^^^^^^^^^^^^^
 
-Those instructions don't use a database to store artifacts. This means that
-every time Hawkbit will be restarted, its rollout campaigns will be lost. This
-is handy for a development environment but unsustainable for a real world
-scenario. Please refer to `the Hawkbit documentation`_ for details on how to set it
-up with a MariaDB database.
+Persistent storage
+""""""""""""""""""
+
+Those instructions don't use a database to store artifacts and metadata. This
+means that every time Hawkbit will be restarted, its rollout campaigns will be
+lost. This is handy for a development environment but unsustainable for a real
+world scenario.
+
+You can set up a MariaDB server to keep data between two executions of Hawkbit.
+Start by installing the `mariadb-server` package from your distribution's
+repositories. Then, make sure the server is running
+
+.. code-block:: bash
+
+    $ systemctl start mariadb-server
+
+Once MariaDB is running, you need to create a database for Hawkbit. For the
+rest of the instructions, we will use the default MariaDB user whose username
+is *root* and password is empty but you can create a new user and adapt the
```

Comment:
Are we saying here that the recommended way of running MariaDB is with the user root?